pkg/types/report.go

```diff
@@ -51,10 +51,10 @@ const (
 	ClassCustom      = "custom"
 
 	// ComplianceNsa is the compliance checks for nsa
-	ComplianceNsa      = Compliance("nsa")
-	ComplianceCIS      = Compliance("cis")
-	ComplianceAWSCIS12 = Compliance("awscis1.2")
-	ComplianceAWSCIS14 = Compliance("awscis1.4")
+	ComplianceNsa      = Compliance("k8s-nsa-1.0")
```

Comment:
Isn't it 1.2? Does it mean we don't support 1.2, but 1.0?
https://media.defense.gov/2022/Aug/29/2003066362/-1/-1/0/CTR_KUBERNETES_HARDENING_GUIDANCE_1.2_20220829.PDF

```suggestion
	ComplianceNsa      = Compliance("k8s-nsa-1.2")
```

Reply:
when we implemented (in starboard) and now its for 1.0 , there are minor changes between 1.2 to 1.0 we could revise it to 1.2 , maybe we can do it later

Reply:
note they are not keeping only latest spec

Reply:
Do we want to keep all the minor versions?

- k8s-nsa-1.0
- k8s-nsa-1.1
- k8s-nsa-1.2
- (k8s-nsa-1.3)

If no, what if we change it to `k8s-nsa`?

Reply:
sound good to keep it `k8s-nsa` in cli but in docs and defSec we should specified that its 1.0 until we update it